src/ostorlab/serve_app/types.py

```diff
@@ -342,3 +342,55 @@ class OxoScansType(graphene.ObjectType):
     """Graphene object type for a list of scans."""
 
     scans = graphene.List(OxoScanType, required=True)
+
+
+class AgentArgumentInputType(graphene.InputObjectType):
+    """Input object type for an agent argument."""
+
+    name = graphene.String(required=True)
+    type = graphene.String(required=True)
+    description = graphene.String(required=False)
+    value = common.Bytes(required=False)
+
+
+class AgentGroupAgentCreateInputType(graphene.InputObjectType):
```

Comment:
```suggestion
class AgentGroupAgentInputType(graphene.InputObjectType):
```

Reply:
this is named `AgentGroupAgentCreateInputType` to align with RE